wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -340,6 +341,11 @@ public WorkspaceStatus getStatus(String workspaceId) {
           createInternalEnvironment(
               environment, workspaceConfig.getAttributes(), workspaceConfig.getCommands());
       RuntimeContext runtimeContext = infrastructure.prepare(runtimeId, internalEnv);
+
+      internalEnv.getMachines()
+              .values()
+              .forEach(internalMachineConf -> internalMachineConf.getAttributes().put(CONTAINER_TYPE_ATTRIBUTE, "user-container"));
```

Comment:
It would be better to make value a constant too

Reply:
Done.